Automattic/jetpack
# Component: Add Pricing Slider component

Fixes #31345 

## Proposed changes:

* Add component `Pricing Slider` with third-party library [ReactSlider](https://zillow.github.io/react-slider/#reactslider).
* Add storybook use cases for operations.

### Other information:


## Jetpack product discussion



p1HpG7-nar-p2

## Does this pull request change what data or activity we track or use?



N/A.

## Testing instructions:






* Run the local storybook with `cd projects/js-packages/storybook && npm run storybook:dev`.
* Ensure the `Pricing Slider` component under `JS Packages` works well.

<img width="661" alt="截圖 2023-06-19 下午12 04 03" src="https://github.com/Automattic/jetpack/assets/6869813/4ac125d0-99f6-4f71-aa44-3c0ec9b2d0d5">
